tools/packaging/kernel/configs/fragments/arm64/base.conf

```diff
@@ -43,7 +43,6 @@ CONFIG_ARM64_RAS_EXTN=y
 # ARMv8.5 architectural features
 #
 CONFIG_ARCH_RANDOM=y
-CONFIG_RANDOM_TRUST_CPU=y
```

Comment:
Since we had this enabled, should the kernel param be passed for all hypervisors?

Reply:
Err, IMO, CPU RNG is a substitution for RNG device. It seems only DB lack of RNG device, I think we can only pass the parameter for DB.

Reply:
ok, thanks for clarifying.